index.html

```diff
@@ -143,5 +143,18 @@ <h1 class="heading-color headline-text">About Myself</h1>
             <i class="fas fa-chevron-right fa-xs"></i>
         </div>
     </section>
+
+    <div class="fake-background">
+        <section class="contact-section">
+            <h1>Contact me</h1>
```

Comment:
- [ OPTIONAL] Kindly change the `h1` tag into` h2` .... `h6`, I always the best practice to only have one h1 in your index.html